### ISSUE TITLE
amino acid metabolic process/ cell cycle regulation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2426]
This paper has a lot of annotation, and most should probably only be annotated as phenotypes are probably very indirect. I might come back to these later with mnofre info. Hopwever I don't see any experiments demonstrating either of these:

Mecp2 | methyl CpG binding protein 2 |   | positive regulation of cell population proliferation |   | RGD | Rattus norvegicus | ISO | RGD:1349232 | 5-methylcytosine g/t mismatch-specific dna glycosylase pthr15074 | gene |   | PMID:25527496RGD:1624291 | 20170318

  | Mecp2 | methyl CpG binding protein 2 |   | positive regulation of G2/M transition of mitotic cell cycle |   | RGD | Rattus norvegicus | ISO | RGD:1349232 | 5-methylcytosine g/t mismatch-specific dna glycosylase pthr15074 | gene |   | PMID:25527496RGD:1624291 | 20170318

MECP2 | Methyl-CpG-binding protein 2 |   | positive regulation of cell population proliferation |   | CAFA | Homo sapiens | IMP |   | 5-methylcytosine g/t mismatch-specific dna glycosylase pthr15074 | protein |   | PMID:25527496 | 20170116

MECP2 | Methyl-CpG-binding protein 2 |   | positive regulation of G2/M transition of mitotic cell cycle |   | CAFA | Homo sapiens | IMP |   | 5-methylcytosine g/t mismatch-specific dna glycosylase pthr15074 | protein |   | PMID:25527496 | 20170116

and again the identical experimental codes are used  for rat and human. A Which organism (s) were these experiments performed in?

**Comment by ValWood** [comment]
This is a stretch based on these data.

Mecp2 | methyl CpG binding protein 2 |   | positive regulation of microtubule nucleation |   | RGD | Rattus norvegicus | ISO | RGD:1349232 | 5-methylcytosine g/t mismatch-specific dna glycosylase pthr15074 | gene |   | PMID:25527496RGD:1624291 | 20170318


For now "spindle organization" is probably overstretching the date , but I  will leave that one for now. I will probably address this in more detail later with reasons why it isn't directly involved in this process.

**Comment by ValWood** [comment]
raised a dispute for. the CAFA annotation

**Comment by Antonialock** [comment]
I am a bit unsure what to change the annotation to. 

from a more recent paper:
 Indeed, nowadays MeCP2 appears as a multifunctional protein that manifests different activities depending on its partners and post‐translational modifications...we have recently demonstrated that MeCP2 is functionally associated with centrosomes and, similarly to many centrosomal proteins, its deficiency causes aberrant spindle geometry, defects in cell proliferation, microtubule nucleation and prolonged mitosis. This novel function reconciled previous data regarding a role of MeCP2 in cell growth, cytoskeleton stability, and axonal transport 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7278541/

it must have some real moonlighting function at the centrosome since it localizes there?

**Comment by ValWood** [comment]
Not sure, leave it if you think its sound. 
Even if it has a centrosomal, a role in "cell cycle regulation" would be a stretch.  That seems that it would be a phenotype, defect in spindle/ stuffs up cell cycle. 

Cell population proliferation should be removed (that is only for use with for controlling proliferation of specific cell types ) - I don't think it should really be in GO

**Comment by Antonialock** [comment]
this might be a very stupid question, but I read that mecp2 is enriched at the centromere, could it just be anchored to the centrosome since it's centromeric? At least in cerevisiae, the kinetochore goes to the SPB during most of the cell cycle and only comes off to replicate centromeric DNA in S phase (but are then recaptured) - I don't know much about this

**Comment by Antonialock** [comment]
"In vivo MeCP2 was shown to be enriched at pericentric heterochromatin [[4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7226738/#B4-cells-09-00878)]. Pericentric heterochromatin is localized in proximity to the centromere and enriched in AT-rich major satellite DNA repeats occupying about 10% of the mouse genome"
-https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7226738/

**Comment by Antonialock** [comment]
in which case it could probably be linked back to its function at the chromatin level?

**Comment by ValWood** [comment]
Yeah I'm not convinced it is meaningful.  All of the phenotypes could be explained by gene expression defects. But I know nothing about this particular either. I think I would want to see come real connection to a centrosomal function before I made any annotations around this observation.
